src/daemon/daemon.cpp

```diff
@@ -2168,8 +2169,11 @@ try
         }
         auto& vm = it->second;
 
-        if (mp::utils::invalid_target_path(q_target_path))
+        if (MP_UTILS.invalid_target_path(q_target_path))
         {
+            mpl::log(mpl::Level::error,
```

Comment:
Does this need to be an error? It makes the message in the CLI a little pedantic:
```
% multipass mount . foo:/home/ubuntu
[2025-09-25T19:26:02.748] [error] [daemon] Invalid target path: /home/ubuntu
mount failed: The following errors occurred:
unable to mount to "/home/ubuntu"
```

Reply:
The more I think about this, the more I get convinced that, by automatically sending logs to users, we're conflating purposes/audiences...

Reply:
Done.